SelectBox: update the 'fieldTemplate' property (#3912)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md

A template name or container.

---

> If you define a **fieldTemplate**, the component does not render the underlying HTML. In this case, manually render hidden input with the corresponding name attribute to submit values through HTML form.

Reviewer: ```suggestion
If you define a **fieldTemplate**, the component does not render the underlying HTML. In this case, manually render hidden input with the corresponding name attribute to submit values through a HTML form.
```

> If you define a **fieldTemplate**, the component does not render the underlying HTML. In this case, manually render hidden input with the corresponding name attribute to submit values through a HTML form.

Reviewer 2: ```suggestion
If you define a **fieldTemplate**, the component does not render the underlying HTML. In this case, you should render hidden input with the corresponding name attribute to submit values through a HTML form.
```

> The example below shows how to define the **fieldTemplate**. If you want to customize field text, use the TextBox [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.

Reviewer: ```suggestion
The example below shows how to define the **fieldTemplate**. If you want to customize field text, use the TextBox's [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.
```

> The example below shows how to define the **fieldTemplate**. If you want to customize field text, use the TextBox's [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.

Reviewer 2: `to customize field text` A field? or fields in general?

Author: A component (SelectBox or TagBox) has a field, I'm talking about such components in general.

Reviewer 2: ```suggestion
The example below shows how to define the **fieldTemplate**. If you want to customize this field's text, use the TextBox's [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.
```

> The example below shows how to define the **fieldTemplate**. If you want to customize this field's text, use the TextBox's [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.

Author: ```suggestion
The example below shows how to define the **fieldTemplate**. If you want to customize the field's text, use the TextBox's [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.
```
